patches/minecraft/net/minecraft/nbt/NBTTagList.java.patch

```diff
@@ -9,9 +9,92 @@
  {
      private static final Logger field_179239_b = LogManager.getLogger();
      private List<NBTBase> field_74747_a = Lists.<NBTBase>newArrayList();
-@@ -281,4 +281,5 @@
+@@ -281,4 +281,88 @@
      {
          return this.field_74746_b;
      }
-+    @Override public java.util.Iterator<NBTBase> iterator() {return field_74747_a.iterator();}
++
++    // Forge START
++
++    @Override
++    public java.util.Iterator<NBTBase> iterator()
++    {
++        return field_74747_a.iterator();
++    }
++
++    public byte getByte(int i)
++    {
++        if (i >= 0 && i < this.field_74747_a.size())
++        {
++            NBTBase nbtbase = this.field_74747_a.get(i);
++            if (nbtbase.func_74732_a() == 1)
```

Comment:
No need to hardcode these constants in our own code, use `Constants.NBT` instead